ext/intl/bidi/bidi.c

```diff
@@ -228,15 +273,17 @@ static PHP_METHOD(IntlBidi, setContext) {
 	UChar *uprologue = NULL, *uepilogue = NULL;
 	int32_t uprologue_len = 0, uepilogue_len = 0;
 
-	if (zend_parse_parameters_throw(ZEND_NUM_ARGS(), "|S!S!", &prologue, &epilogue) == FAILURE) {
-		return;
-	}
+	ZEND_PARSE_PARAMETERS_START(0, 2)
+		Z_PARAM_OPTIONAL
+		Z_PARAM_STR_EX(prologue, 0, 0)
+		Z_PARAM_STR_EX(epilogue, 0, 0)
```

Comment:
Ditto for this line and its arginfo as well, of course.

Reply:
Was already fixed due to a previous fix.